.licenserc.yaml

```diff
@@ -14,6 +14,7 @@ header:
 
   paths-ignore:
     - "KubeArmor/BPF/*"
+    - "KubeArmor/**/*bpf*.go"
```

Comment:
wondering why ignoring those files?

Reply:
These files are auto generated by https://github.com/cilium/ebpf/tree/master/cmd/bpf2go

I couldn't find a way to modify them and add license headers while generating